polkadot/cli/src/cli.rs

```diff
@@ -152,6 +152,12 @@ pub struct RunCmd {
 	/// explicitly advised to.
 	#[arg(long)]
 	pub enable_approval_voting_parallel: bool,
+
+	/// How long finalized data should be kept in the availability store (in hours).
+	/// Only used for testnets. If not specified, set to 1 hour. Always set to 25 hours for live
```

Comment:
I would prefer a default value for live networks, with the ability to change it only for testnets. This way, we can explicitly set the shorter time in the CLI.